packaging/nnstreamer.spec

```diff
@@ -117,6 +121,18 @@ BuildRequires: lcov
 %if 0%{mvncsdk2_support}
 BuildRequires:	pkgconfig(libmvnc)
 %endif
+
+# for Vivante
+%if 0%{?vivante_support}
+BuildRequires:  pkgconfig(opencv)
+BuildRequires:  pkgconfig(ovxlib)
+BuildRequires:  pkgconfig(amlogic-vsi-npu-sdk)
+BuildRequires:  pkgconfig(dann)
```

Comment:
Do you need dann as well?

Reply:
Let's keep the "**pkgconfig(dann)**"  statement for them temporarily.

Reply:
Instead, I added a "TODO" statement.